sunpy/time/tests/test_time.py

```diff
@@ -288,9 +288,9 @@ def test_parse_time_astropy_formats(ts, fmt):
 def test_parse_time_int_float():
     # int and float values are not unique
     # The format has to be mentioned
-    with pytest.raises(ValueError):
+    with pytest.raises(ValueError, match="Input values did not match any of the formats where the format keyword is optional:\n- 'datetime': Input values for datetime class must be datetime objects\n- 'ymdhms': input must be dict or table-like\n- 'iso': Input values for iso class must be strings\n- 'isot': Input values for isot class must be strings\n- 'yday': Input values for yday class must be strings\n- 'datetime64': Input values for datetime64 class must be datetime64 objects\n- 'fits': Input values for fits class must be strings\n- 'byear_str': Input values for byear_str class must be strings\n- 'jyear_str': Input values for jyear_str class must be strings\n- 'astropy_time': Input values for astropy_time class must all be the same astropy Time type."):
```

Comment:
Don't list all of the formats. Just catch the start of the error message.

Reply:
Done. Thank You.